intercom-java/src/main/java/io/intercom/api/Intercom.java

```diff
@@ -5,8 +5,6 @@
 public class Intercom {
 
     static class Context {
-        private volatile AuthKeyType authKeyType = AuthKeyType.API_KEY;
-        private volatile String apiKey;
         private volatile String token;
         private volatile String appID;
```

Comment:
I think we can also remove `appId` here. It was used in conjunction with `apiKey` when generating the Auth header. Now we just need the `token`

Reply:
Done 👍